code/datums/ai/basic_mobs/basic_subtrees/mine_walls.dm

```diff
@@ -0,0 +1,73 @@
+//behavior to find mineable mineral walls
+
+/datum/ai_planning_subtree/mine_walls
+	var/find_wall_behavior = /datum/ai_behavior/find_mineral_wall
```

Comment:
Would `/datum/element/wall_smasher` be considered a required operational datum or do you think this could work without it?

Reply:
it could work if the mob uses projectiles to mine, like the minebot

Reply:
mkay